index.html

```diff
@@ -189,7 +188,7 @@ <h1 class="professional-h1">Professional Art<br />Printing Data</h1>
           <li class="listons second">bootstrap</li>
           <li class="listons third">ruby</li>
         </ul>
-        <button class="liston-button bb1">See Project</button>
+        <button class="liston-button bb1 cproject">See Project</button>
```

Comment:
- [ ] Add responsive design for your popup, as you can see in the image below, when you are in the mobile version the popup can look better to show the information properly.

![image](https://user-images.githubusercontent.com/60164239/131806290-afc3fe21-b8b7-4f46-b863-cb8b28b787d0.png)